src/theme/index.hbs

```diff
@@ -256,6 +256,13 @@
         <script src="{{ path_to_root }}searcher.js" type="text/javascript" charset="utf-8"></script>
         {{/if}}
 
+        {{#if offline_support}}
+        <script type="text/javascript">
+            // Must insert before loading book.js
+            window.sevice_worker_script_src = "{{ path_to_root }}sw.js";
```

Comment:
BTW, this seems to be misspelled?  That is `sevice` → `service` with an `r`.  If that is not intentional, can all the instances be updated?